### PR TITLE
fix setIcon using svg

### PR DESCRIPTION
**Pull request by ArtificialOwl**

### CODE REVIEW

File: lib/Activity/Provider.php [hunk 1]
```diff
@@ -93,10 +93,7 @@ private function initActivityParser(IEvent $event, array $params): void {
 	 * @param IEvent $event
 	 */
 	private function setIcon(IEvent $event): void {
-		$path = $this->urlGenerator->imagePath(
-			Application::APP_ID,
-			'circles.' . $this->activityManager->getRequirePNG() ? 'png' : 'svg'
```
Comment (artonge) [inline lib/Activity/Provider.php:98]: Why is the check not needed anymore?

Comment (ArtificialOwl) [reply lib/Activity/Provider.php:98]: we have no more png anymore since 22, and it seems to be fine having svg for the generated event

**Review by artonge — APPROVED** [review]

**Comment by mejo-** [comment]
stable29 and stable28 don't need backports?

**Comment by blizzz** [comment]
> stable29 and stable28 don't need backports?

This went against stable28, newer versions are already adjusted.

**Pull request merged**